decoder.go

```diff
@@ -54,6 +55,15 @@ func (d *Decoder) IgnoreUnknownKeys(i bool) {
 	d.ignoreUnknownKeys = i
 }
 
+// UnderscoreToCamelCase allows automatically map form values named
+// in "underscore" style to struct fields with "CamelCase" naming
+// convention.
+//
+// The default value is false.
+func (d *Decoder) UnderscoreToCamelCase(u bool) {
```

Comment:
I would just make this `UnderscoreToCamelCase()` - calling this should automatically set the underlying struct field to `true`. Calling it with false doesn't make much sense.